source/features/parse-backticks.tsx

```diff
@@ -1,15 +1,17 @@
 import './parse-backticks.css';
-import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
+import {observe} from 'selector-observer';
 
 import features from '.';
-import observeElement from '../helpers/simplified-element-observer';
 import {parseBackticks} from '../github-helpers/dom-formatters';
 
 function parse(selectors: string[]): void {
-	for (const element of select.all(selectors.map(selector => selector + ':not(.rgh-backticks-already-parsed)'))) {
-		element.classList.add('rgh-backticks-already-parsed');
-		parseBackticks(element);
+	for (const selector of selectors) {
+		observe(selector, {
```

Comment:
```suggestion
	observe(selectors.join(','), {
```

I'm pretty sure `rgh-backticks-already-parsed` is still needed; last time I tried that was exactly the issue. Try for example:
1. load this PR
2. visit another PR tab (to trigger AJAX navigation)
3. click the browser's back button

You might see that the feature run twice on the same piece of DOM

Reply:
Also you can merge all the `init`s into one so there's a single `observe` for the whole feature

Reply:
> 3. click the browser's back button

Usually there need to be a 4th step to trigger that: then click the forward button. But whatever.